AppCenter/AppCenterTests/MSEncrypterTests.m

```diff
@@ -17,80 +19,411 @@ @implementation MSEncrypterTests
 - (void)setUp {
   [super setUp];
   self.keychainUtilMock = [MSMockKeychainUtil new];
-  self.keyTag = @"kMSTestEncryptionKeyTag";
 }
 
 - (void)tearDown {
   [self.keychainUtilMock stopMocking];
-  [MSEncrypter deleteKeyWithTag:self.keyTag];
+  [MSMockKeychainUtil clear];
```

Comment:
Why do you need this one? `[self.keychainUtilMock stopMocking];` should be enough.

Reply:
Are you saying that [MSEncrypter deleteKeyWithTag:self.keyTag]; was not needed before then?

Reply:
Sorry for confusion. I'm talking about `[MSMockKeychainUtil clear];`.

Reply:
But before, you can see that it was doing `[self.keychainUtilMock stopMocking];` *and* `[MSEncrypter deleteKeyWithTag:self.keyTag];`, so i'm wondering why that was the case?

Reply:
Ah yes. The test class doesn't have `NSUserDefaults` mock so it it will actually using a real one. Let's keep it.